storage/storage.go

```diff
@@ -456,7 +456,7 @@ func listBlockDevices(userDefined []*BlockDevice) ([]*BlockDevice, error) {
 		return nil, err
 	}
 
-	if err = utils.VerifyRootUser(); err == nil {
+	if errString := utils.VerifyRootUser(); errString == "" {
```

Comment:
With the early check, do we need this check at all?

Reply:
Maybe not? I was concerned there was some other code flow which used storage before the first check. You think I should drop it?

Reply:
Yep, please drop it.

Reply:
Dropping. First fall out is one of the storage tests is now failing when not run as root. Updating...

Reply:
@dorileo Now now the coverage numbers of storage have dropped. :(

Reply:
@dorileo Updates made as we discussed offline.
New changes are pushed and testing now.